plugins/adr/src/translations.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { createTranslationRef } from '@backstage/core-plugin-api/alpha';
+
+/** @alpha */
+export const adrTranslationRef = createTranslationRef({
```

Comment:
I think this needs to be exported from this package so consumers can modify the translations? It looks like the current convention is to ship it via [`alpha` exports](https://github.com/backstage/backstage/pull/17436/files#diff-6545ef79075af1d48222e636187d3bf1e24e825926c0e98357300ed8dc22c6ab) as this is still experimental

Reply:
Do I miss something? I thought line 18 marks the export as `alpha`, isn't it?

Reply:
Yes - marking it `@alpha` generates an alpha export for the `api-report` (which reminds me, you will also need to update the `api-reports.md` by running `yarn build:api-reports` from the repo root, after you make the changes below) 

I meant that you need to actually export this symbol from the package itself - for example see the change [here](https://github.com/backstage/backstage/pull/17436/files#diff-6545ef79075af1d48222e636187d3bf1e24e825926c0e98357300ed8dc22c6ab) which exports things from an `alpha.ts` file and also the associated [changes here](https://github.com/backstage/backstage/pull/17436/files#diff-e1b268e9d36dc2d5ad582a461567cd08f5807b61457a282a414490323c77ad5fR8-R24) you would also need to make to export these alpha symbols from this package